test/e2e/tests/dapp-interactions/permissions.spec.js

```diff
@@ -17,7 +17,7 @@ describe('Permissions', function () {
       },
       async ({ driver, localNodes }) => {
         const addresses = await localNodes[0].getAccounts();
-        const publicAddress = addresses[0];
+        const publicAddress = addresses[0].toLowerCase();
```

Comment:
we need to lower case the acontract address as in anvil we get the checksummed value instead